rsql_core/src/executors/sql.rs

```diff
@@ -67,9 +67,14 @@ impl<'a> SqlExecutor<'a> {
         Span::current().pb_set_style(&ProgressStyle::with_template(
             "{span_child_prefix}{spinner}",
         )?);
-        let command = if sql.len() > 6 { &sql[..6] } else { "" };
+        let is_select = if let Some(query_meta) = self.connection.parse_sql(sql) {
+            matches!(query_meta, QueryMeta::Query)
+        } else {
+            let command = if sql.len() > 6 { &sql[..6] } else { "" };
+            command.to_lowercase() == "select"
+        };
```

Comment:
Is this still required?  Can we require that the logic only exists in `parse_sql()` to determine `QueryMeta::Query`

Reply:
I can move it into parse_sql as the fallback option.

I kept it in place so queries that sqlparser has trouble with ultimately see no change in behavior compared to previous versions of rsql (I trust underlying db engines will have more complete query parsers than sqlparser will)